Documentation suggestions

redapple opened the pull request:
Submitting early to get feedback.
I have not reviewed all sections yet though.

vshlapakov commented on docs/client/overview.rst:
```diff
@@ -1,32 +1,34 @@
 
-Or check the projects summary::
+.. note::
+    ``.list()`` does not return :class:`~scrapinghub.client.projects.Project`
```
:+1:

vshlapakov commented on docs/client/overview.rst:
```diff
@@ -91,116 +88,145 @@ To select a particular spider to work with::
-:class:`~scrapinghub.client.jobs.Jobs` collection is available on project/spider
-level.
+Essentially, the purpose of spiders is to be run in Scrapinghub's platform.
+And a collection of jobs is represented by a :class:`~scrapinghub.client.jobs.Jobs`
```
Maybe link spiders and jobs here somehow? Maybe `a collection of spiders jobs`?

vshlapakov commented on docs/client/overview.rst:
```diff
@@ -91,116 +88,145 @@ To select a particular spider to work with::
 
-count
-^^^^^
+To select a specific job for a project, use ``.jobs.get(<jobID>)``::
```
I'd replace `jobID` with `jobKey` or `JobKey`: my worries that it can be mixed with job id that is a part of job key, based on the concept that jobkey consists of 1) project id, 2) spider id and 3) job id.

vshlapakov commented on docs/client/overview.rst:
```diff
@@ -282,12 +308,14 @@ To remove existing tag ``existing`` for all spider jobs::
 
-Job
----
+Individual Job's data
```
`Individual job's data`? (I mean capitalization)

vshlapakov commented on docs/quickstart.rst:
```diff
@@ -14,43 +14,55 @@ The quick way::
 
-You can also install the library with `MessagePack`_ support, it provides better
-response time and improved bandwidth usage::
+You can also install the library with `MessagePack`_ support (recommended),
```
`It's recommended to install the library with ..., it provides ..`?

vshlapakov commented:
@redapple Amazing work, Paul, please go on :+1:  I can only nitpick a bit :blush:

redapple commented:
Thanks for the feedback @vshlapakov . I edited more sections. I hope I addressed your points.

I was wondering about "Samples": I'm not sure it deserves a section in the Overview. I don't expect many users to use them. Maybe the API section on Samples is sufficient.

vshlapakov commented on docs/client/overview.rst:
```diff
@@ -116,34 +117,37 @@ or a :class:`~scrapinghub.client.spiders.Spider` instance respectively.
     >>> job = spider.jobs.run()
 
-You can also use ``run`` at the project level, the difference being that
+You can also use ``.run()`` at the project level, the difference being that
```
Same `.jobs.run()` for consistency?

vshlapakov commented on docs/client/overview.rst:
```diff
@@ -426,6 +509,11 @@ for a job::
 
+.. note::
+    The term "samples" is not about providing samples of the jobs data.
+    It's more about job runtime job statistics, that your job can send
```
`job runtime statistics`?

vshlapakov commented:
@redapple Left a couple of comments, but overall looks good, thank you.

I don't mind to drop Samples section from overview, I have never used it by myself and haven't heard if someone hardly uses it, makes sense to remove it from Overview.

redapple commented:
@vshlapakov , I removed the "Samples" section.
I'm tempted to leave Frontier and Settings review to another PR, as this one is already quite a change.

redapple commented:
@vshlapakov , I'll leave this PR as is. Frontier and Settings can be improved in another round I think.

vshlapakov commented on docs/client/overview.rst:
```diff
@@ -431,9 +543,25 @@ Or post multiple events at once::
+They’re often used by Scrapinghub projects as a single place to write
+information from multiple scraping jobs.
+
+Read more about _Collections_ `in the official docs`_.
```
Tiny issue: `Collections` word is printed as `_Collections_`, is it intended?

redapple replied:
Good catch. Updated.

vshlapakov commented:
Many thanks @redapple, please squash it and I'll merge the PR.

redapple commented:
@vshlapakov done.
